draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -386,20 +509,24 @@ The Issuer creates Disclosures first for the sub-claims and then includes their
 
 {{examples/address_only_recursive/disclosures.md}}
 
-### Hash Function Claim {#hash_function_claim}
+## Hash Function Claim {#hash_function_claim}
 
-The claim `_sd_alg` indicates the hash algorithm
-used by the Issuer to generate the digests over the salts and the
-claim values. If the  `_sd_alg` claim is not present, a default value of `sha-256` is used.
+The claim `_sd_alg` indicates the hash algorithm used by the Issuer to generate
+the digests over the salts and the claim values. If the  `_sd_alg` claim is not
```

Comment:
```suggestion
The claim `_sd_alg` indicates the hash algorithm used by the Issuer to generate
the digests as described in (#creating_disclosures). If the  `_sd_alg` claim is not
```
to avoid saying when the hashed array contains a claim name and when it does not